next.config.js

```diff
@@ -22,4 +22,18 @@ module.exports = {
       process.env.PUBLIC_BACKEND_URL || process.env.BACKEND_URL || "",
     UI_VERSION,
   },
+  async redirects() {
+    return [
+      {
+        source: "/",
+        destination: "/search",
+        permanent: true,
+      },
+      {
+        source: "/index",
+        destination: "/_index",
+        permanent: true,
+      },
+    ];
+  },
```

Comment:
Not familiar with the `redirects` function in the config. With this, do you still need the `useEffect` in page/index.js? The NextJS docs are not remotely clear about this.